tools/cfn2ts/lib/codegen.ts

```diff
@@ -322,8 +322,12 @@ export default class CodeGenerator {
      * Since resolve() deep-resolves, we only need to do this once.
      */
     private emitCloudFormationPropertiesOverride(propsType: genspec.CodeName) {
-        this.code.openBlock('protected renderProperties(): { [key: string]: any } ');
-        this.code.line(`return ${genspec.cfnMapperName(propsType).fqn}(${CORE}.resolve(this.properties));`);
+        this.code.openBlock(`public get propertyOverrides(): ${propsType.className}`);
```

Comment:
had to be learning in a review, but this is how all the L1s are getting the strongly typed `propertyOverrides` object without the team having to go modify each resource right?

Reply:
We can't even go and modify "each resource", because we they're all code-generated. This is the code generator.

Reply:
Yap!